rosetta-source/src/main/java/cdm/product/template/processor/CommodityClassificationMetaMappingProcessor.java

```diff
@@ -25,12 +25,6 @@ public void map(Path synonymPath, List<? extends RosettaModelObjectBuilder> buil
         if (!forwardPayoutBuilders.isEmpty()) {
             ForwardPayout.ForwardPayoutBuilder forwardPayoutBuilder = forwardPayoutBuilders.get(0);
             // create reference
```

Comment:
the code to create the reference has been removed so I assume this comment should be removed too?